src/meal-details.js

```diff
@@ -0,0 +1,96 @@
+/* eslint-disable import/prefer-default-export */
+import { postComment, getComments } from './involvement';
+
+const mealDetailsContent = document.querySelector('.meal-details-content');
+const recipeCloseBtn = document.getElementById('recipe-close-btn');
+
+// get details of the meal
+export const getMealDetails = async (mealId) => {
+  await fetch(`https://www.themealdb.com/api/json/v1/1/lookup.php?i=${mealId}`)
+    .then((response) => response.json())
+    // eslint-disable-next-line no-use-before-define
+    .then((data) => mealDetailsModal(data.meals[0]));
+};
+
+// create a modal
+function mealDetailsModal(meal) {
+  const ingredient = [];
+  const measure = [];
+  let i = 1;
+  while (meal[`strIngredient${i}`]) {
+    ingredient.push(meal[`strIngredient${i}`]);
+    measure.push(meal[`strMeasure${i}`]);
+    i += 1;
+  }
+  let ingredientsHTML = '';
+  // eslint-disable-next-line no-plusplus
+  for (let i = 0; i < ingredient.length; i++) {
```

Comment:
- [ ] Instead of suppressing the eslint check you can easily solve this :arrow_down: 
```suggestion
  // eslint-disable-next-line no-plusplus
  for (let i = 0; i < ingredient.length; i+=1) {
```